pctiler/pctiler/endpoints/item.py

```diff
@@ -40,151 +31,51 @@ class MapParams:
     item: str = Query(..., description="STAC Item ID")
 
 
-@dataclass
-class ItemTileFactory(MultiBaseTilerFactory):
-    """Item tiler
-
-    Contains titiler routes for STAC Items, along with some
-    custom routes.
-    """
-
-    reader: Type[MultiBaseReader] = ItemSTACReader
-
-    colormap_dependency: Callable[..., Optional[Dict]] = PCColorMapParams
-
-    path_dependency: Callable[..., str] = ItemPathParams
-
-    router_prefix: str = get_settings().item_endpoint_prefix
-
-    templates = Jinja2Templates(directory="/opt/src/templates")
-
-    def register_routes(self) -> None:
-        self.register_map()
-
-        super().register_routes()
-
-        self.register_poly_crop()
-
-    def register_map(self) -> None:
-        """Register /map endpoint"""
-
-        ############################################################################
-        # Map: Rendered map of Item
-        ############################################################################
-
-        @self.router.get("/map", response_class=HTMLResponse)
-        def map(
-            request: Request, map_params: MapParams = Depends(MapParams)
-        ) -> Response:
-
-            collection_id, item_id = (map_params.collection, map_params.item)
+pc_tile_factory = MultiBaseTilerFactory(
+    reader=ItemSTACReader,
+    path_dependency=ItemPathParams,
+    colormap_dependency=PCColorMapParams,
+    router_prefix=get_settings().item_endpoint_prefix,
+)
```

Comment:
There is no more need to create a custom TilerFactory, we can use `MultiBaseTilerFactory` and set custom params